packages/account/src/Constants/routes-config.ts

```diff
@@ -152,12 +155,12 @@ const initRoutesConfig = () => [
     },
 ];
 
-let routesConfig;
+let routesConfig: TRouteConfig[] | undefined;
```

Comment:
```suggestion
let routesConfig: TRouteConfig[] | undefined;
```
```suggestion
const routesConfig: TRouteConfig[] = [];
```
it's gonna be GG if our routes is undefined. Set it to const because it is not re-initialized and u guys used push to add new routes

Reply:
default value of `[]` is given for this variable.
later in the code, i can see this variable being reinitialized. so kept it a `let` 👍🏼

Reply:
update: empty array is causing issue on initial load of the accont settings page. hence kept it undefined.
by the way, whats GG  @yashim-deriv ?